css/blog.css

```diff
@@ -0,0 +1,104 @@
+@import url(https://fonts.google.com/specimen/Raleway);
```

Comment:
Can we use the built in fonts we are using throughout the site pls?

Reply:
What's the problem with this? Same fonts. Didn't?

Reply:
Can't we go for a single font per the entire website.